### DISCUSSION TITLE
Why is this project called "YALMIP"?

### DISCUSSION THREAD

**Discussion by Misrandeer** [yalmip/YALMIP#1329] [General discussion] [answered]
I want to know why is this project called "YALMIP"?
Is it some words' abbreviation?
What are those words?
Thank you!

**Comment by johanlofberg** [comment, marked as answer]
In ancient times when dinosaurs roamed the streets it was an acronym for Yet another LMI parser, but now it is just a name
https://yalmip.github.io/faq/whyyalmip/
